R/zchunk_L254.transportation_USA.R

```diff
@@ -88,100 +113,461 @@ module_gcam.usa_L254.transportation_USA_DISABLED <- function(command, ...) {
     L154.out_mpkm_state_trn_nonmotor_Yh <- get_data(all_data, "L154.out_mpkm_state_trn_nonmotor_Yh")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses repeat_and_add_vector
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+    # Delete transportation sectors in the full USA region
+    L254.Supplysector_trn %>%
+      filter(region == "USA") %>%
+      select(region, supplysector) ->
+      L254.DeleteSupplysector_USAtrn
+
+    # Delete energy final demand sectors in the full USA region
+    L254.PerCapitaBased_trn %>%
+      filter(region == "USA") %>%
+      select(one_of(c(LEVEL2_DATA_NAMES[["EnergyFinalDemand"]]))) ->
+      L254.DeleteFinalDemand_USAtrn
+
+    # This applies to the supplysectors, subsectors, and stub tech characteristics of the states.
```

Comment:
Comment should be clearer--what exactly is `process_USA_to_states` doing, at a high level?